src/cpp/session/resources/dependencies/r-packages.json

```diff
@@ -251,7 +251,7 @@
             "source": false
         },
         "rsconnect": {
-            "version": "0.8.24",
+            "version": "1.3.1",
```

Comment:
Paranoia based on past experience: make sure this package version doesn't depend on R >= 4.0.

Reply:
I indeed did double-check that :-)